src/index.ts

```diff
@@ -29,10 +29,14 @@ export class Uwuifier {
   @InitModifierParam()
   private _exclimationsModifier: number = 1;
 
```

Comment:
You're still initializing it with a number here, even though you also are using the constructor.